source/includes/getting-started/_concepts.html.md

```diff
@@ -1,61 +1,69 @@
+The Bonsai AI Engine also determines when to train each concept, how much (or little) to train each concept based on its relevance, and, ultimately, produces a trained BRAIN. Furthermore, it does something we call **meta-learning**. Meta-learning means that the system keeps a kind of memory or record about each program it's seen, the data it used for training, and the generated AIs that it's made. It also records how fast those AIs trained and how accurate they became. The BRAIN server is continuously learning and improving over that dataset.
 
 ### You might say the system learns to make learning systems.
 
 ## What is a simulator?
 
 A simulator imitates a real world situation in a virtual environment, which enables a user to model and test situations that would often, otherwise be very complicated to set up and test. Your simulator could be a game, such as breakout, or an emulation of a real world process, such as a self driving car. Training with simulators is especially useful for optimization, or for use with complex control problems that can be modeled in a simulator.
 
-**Remember:**  Currently, a simulation runs on your local machine and streams data to the BRAIN server for training. This means your computer needs to remain connected to the Bonsai Artificial Intelligence Engine while your BRAIN is training. If your computer is disconnected from the server, it automatically picks up where it left off when its reconnected. Today, we only support simulators written in python.
+**IMPORTANT:**  Currently, a simulation runs on your local machine and streams data to the BRAIN server for training. This means your computer needs to remain connected to the Bonsai AI Engine while your BRAIN is training. If your computer is disconnected from the server you can reconnect within an hour and it will pick back up where it left off. Otherwise, your training will start on a new version (your old training is still saved to the previous version). If you want to leave your training running overnight, please make sure that your computer will stay connected by turning off sleep functionality. 
+
+[//]: # (1 hour timeout, no auto-reconnect)
 
-To learn more about connecting your own simulator, refer to our SDK.
+Today, we only support simulators written in python. To learn more about connecting your own simulator, refer to our SDK Reference (Coming soon!).
+
+[//]: # (Simulators only in Python, No SDK Reference)
 
 ## What is training?
 
-Training means teaching it to get a specific outcome on a simulator. The training involves using a specific set of concepts, a curriculum, and lessons, which are described in the Inkling file.
+Training means teaching a simulator to get a specific outcome. The training involves using a specific set of concepts, a curriculum, and lessons, which are described in the Inkling file.
```

Comment:
The BRAIN/AI is learning here, not the simulator.
"means teaching **a BRAIN** to reach a specific outcome **in the simulator**.